app/src/components/main/icon/icon.jsx

```diff
@@ -21,13 +21,17 @@ import styles from './icon.scss';
 
 const cx = classNames.bind(styles);
 
-export const Icon = ({ type, onClick }) => <i className={cx('icon', type)} onClick={onClick} />;
+export const Icon = ({ type, onClick, disabled }) => (
+  <i className={cx('icon', type, { disabled })} onClick={disabled ? null : onClick} />
```

Comment:
Let's update it to `button` to make an element interactive.

Reply:
Done.